lib/Notification/Notifier.php

```diff
@@ -79,7 +79,7 @@ public function prepare(INotification $notification, $languageCode) {
 				}
 
 				$notification->setParsedSubject($subject)
-					->setLink($this->url->getAbsoluteURL('index.php/settings/personal'));
+					->setLink($this->url->getAbsoluteURL('index.php/settings/user'));
```

Comment:
So just change it from getAbsoluteURL to getAbsoluteRouteURL? Or more? Could you add a commit for that?